client/methods.js

```diff
@@ -42,21 +46,22 @@ Package['meteor']['Meteor'].sendCoverage = function (callback) {
 /**
 * Usage: Meteor.exportCoverage(null, function(err) {console.log(err)})
 */
-Package['meteor']['Meteor'].exportCoverage = function (type, callback) {
+Meteor.exportCoverage = function (type, callback) {
     var url = type ? '/coverage/export/'+type : '/coverage/export';
     $.ajax({
         method: 'GET',
         url: url,
         success: function(data) {
             try {
                 let result = JSON.parse(data);
-                if (result.type === "success") {
-                    return callback();
+                if (result.type !== "success") {
+                  throw new Error('An unexpected error occurred while trying to export coverage data');
```

Comment:
callback is not called? so the client caller (spacejam, ...) cannot end correctly. Too bad